src/config/core.h

```diff
@@ -21,6 +21,9 @@
 #ifndef CONFIG_CORE_H
 #define CONFIG_CORE_H
 
+// from tag vXXXXXXXX.Y -> XXXXXXXXY
+#define HERCULES_VERSION 202009200
```

Comment:
I'm guessing the 0 at the end of 20200920 is the increment counter of commit of that day =/
why HERCULES_VERSION wants to be different than PACTEVER is anyone guess
but this might do the trick

Reply:
the last number is for multiple releases on the same day eg: `v2020.03.08+2`, `v2020.03.08+1`, `v2020.03.08 `

 and for full list you can check https://github.com/HerculesWS/Hercules/tags

as for why HERCULES_VERSION is not the same as PACKETVER it's because this is the emulator version and not the client version.